_posts/2021-06-04-flavourzyme-a-purified-enzyme-mixture-from-a-oryzae.md

```diff
@@ -0,0 +1,103 @@
+
+Once the upstream process is finished, you end up with a crude liquid containing fungi, enzymes, spent substrate, and all sorts of metabolites.
+
+### Refining the enzymes - downstream processing
+
+In *downstream processing*, we want to refine out the enzymes only. While I couldn't find any literature specifically on the downstream processing of flavourzyme, reading textbook on enzymes was enough to cover the basics [5]. Then, a producer of a Flavourzyme-like product on Alibaba just gave me their HACCP flow diagram, so that made things easy. Here's one example of how it can be done:
+
+#### 1. Filter Press 
+
+The first step is to filter out the mycelia, cell debris, any any other large solids. A *filter press* is used here with perlite, followed by diatomaceous earth, as a filtering aid. The solid matter is captured in the filter aid and removed as filter cake - leaving behind an enzyme mixture. 
+
+![Removing large solids from the raw liquid using perlite and diatomaceous earth ](/assets/images/flavourzyme/Deadendfiltration.png)
+*Schematic of dead-end filtration with a filter aid - elements not to scale*
+
+Note that filtration is the first step because we are dealing with **extracellular** enzymes. If the product was an **intracellular** enzyme, this step would be preceded by a 'disruption' step to break apart cells. 
+
+#### 2. Ultrafiltration Concentration 
+
+At this point, our enzyme mixture is still at a fairly low concentration. Another type of filtration called *ultrafiltration* is used to remove water. By passing the liquid across a filter of the appropriate size (in this case, a filter rated around 10kDa), water will permeate through, leaving behind a more concentrated enzyme solution. This can be repeated until the desired concentration is achieved.
+
+![Removing water from the resulting filtrate](/assets/images/flavourzyme/Ultrafiltration.png)
+*Schematic of cross-flow ultrafiltration - elements not to scale*
+
+#### 3. Final Steps
+
+I won't go into too much detail here, but the final mixture can either be dried by spray-drying with the help of a carrier solid (typically sodium chloride), or packaged as a liquid after passing through a sterilizing filter and adding stabilizers and preservatives.
+
+### The Final Product
+
+If everything's gone well so far we should have a powerful enzyme mixture rated for 500 LAPU/g (one LAPU is the amount of enzyme which hydrolyzes 1 µmol of L-leucine-pnitroanilide per minute - this is one of many enzyme assays for proteolytic enzymes) and between 6% w/w of pure enzyme protein, depending on formulation [6]. By comparison, unrefined shoyu koji contains around 0.2% w/w protease [7].
+
+Novozymes A/S won't sell to non-commercial or non-institutional end-users. So naturally, I'd have to import an imitation product from Alibaba. The other question is, of course, can I brew this at home?
```

Comment:
Can you expand on this conclusion? Did the package arrive? What are the risks of an imitation product? What do you want to make/do with the flavourzyme?

Reply:
I think the conclusion is the weak part of an otherwise excellent article. Think about this more!